src/Sulu/Bundle/WebsiteBundle/Resources/public/js/services/reference-store.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var synonyms = {},
+        references = {},
+
+        resolveAlias = function(alias) {
```

Comment:
Apart from that, why does this code come to the frontend? If we implement it this way, it should at least be normalized on the server already, shouldn't it?